pkgs/development/python-modules/hyperpyyaml/default.nix

```diff
@@ -20,7 +21,15 @@ buildPythonPackage rec {
 
   propagatedBuildInputs = [
     pyyaml
-    ruamel-yaml
+    # Build crashes when using the latest version of ruamel-yaml
+    (ruamel-yaml.overridePythonAttrs (old: rec {
+        version = "0.17.28";
+        src = fetchPypi {
+          pname = "ruamel.yaml";
+          inherit version;
+          hash = "sha256-O/bfHEgdJGOmM75u6G6K7OlBuzKYqaDNbQhl9Hsd3OY=";
+        };
+    }))
```

Comment:
We can't do this in pythonPackages because if an env is created with a different ruamel-yaml then it breaks.

Reply:
Ok, I am observing this right now because `speechbrain` fails to build.
What do you suggest ? Wait for upstream `hyperpyyaml` to adapt to latest `ruamel-yaml` ?